frontend/src/constants/ui/strings.js

```diff
@@ -71,6 +71,7 @@ export default {
     },
     ERROR: {
       userNotFound: "User not found",
+      somethingWentWrong: "Oops, Something went wrong. Please try again later",
```

Comment:
This is not a good error message. I know that we are used to seeing such messages, but they don't offer any useful information to the end user.

Change it to this:
"Sorry, we were unable to process your request. You can try again later or report this issue to bugs@hs-woofer.com".

Reply:
Ack